libbeat/autodiscover/registry.go

```diff
@@ -40,5 +46,7 @@ func NewRegistry() *registry {
 		providers: make(map[string]ProviderBuilder, 0),
 		builders:  make(map[string]BuilderConstructor, 0),
 		appenders: make(map[string]AppenderBuilder, 0),
+
```

Comment:
Nit: Seems like this newline is intentional?

Reply:
done